turbo/rpchelper/filter_id.go

```diff
@@ -0,0 +1,28 @@
+package rpchelper
+
+import (
+	"crypto/rand"
+	"fmt"
+
+	"github.com/ledgerwatch/log/v3"
+)
+
+type (
+	SubscriptionID    string
+	HeadsSubID        SubscriptionID
+	PendingLogsSubID  SubscriptionID
+	PendingBlockSubID SubscriptionID
+	PendingTxsSubID   SubscriptionID
+	LogsSubID         uint64
+)
+
+func generateSubscriptionID() SubscriptionID {
```

Comment:
maybe can replace by `atomic.Uint64.Inc()`

Reply:
I have changed this and added a test